src/main/java/ui/printer/ConsolePrinter.java

```diff
@@ -0,0 +1,32 @@
+package ui.printer;
+
+public class ConsolePrinter implements Printer {
+    private final String GREET_MESSAGE = "계산기 프로그램입니다.\n" +
```

Comment:
현재 java 에서 Multiline에 대한 기능을 제공을 안해주는 상황인 것 같아요.
Adrian Walker의 @Multiline 라이브러리가 있는 것 같은데 필요성이 있다면 사용하셔도 괜찮을 것 같아요!

ex)
```suggestion
/**
계산기 프로그램입니다.
숫자와 사칙연산 기호를 공백으로 구분하여 입력해주세요.
프로그램을 종료하시려면 exit을 입력해주세요
*/
@Multiline String GREET_MESSAGE;
```

Reply:
보기 안좋았는데 그런 라이브러리가 있었군요!
한번 적용해보겠습니다 😄